### PR TITLE
Adjusted checks on clear cb cleanup function to allow for client obje…

### LINKED ISSUE #331: Clean up function of CBC
**Describe the bug**
The clear function for context broker has bug when only use client object as input

**To Reproduce**
```python
    ...
    fiware_header = FiwareHeader(service=SERVICE, service_path=SERVICE_PATH)

    # Create a Context Broker Client
    cb_client = ContextBrokerClient(url=CB_URL, fiware_header=fiware_header)
    clear_context_broker(cb_client=cb_client)
```

**Expected behavior**
- Bug fix
- Implement test to ensure all the cleanup functions can accept both url+fiware_header and client object

### PR DESCRIPTION
**Pull request by djs0109**
closes #331

### CODE REVIEW

File: tests/utils/test_clear.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        self.iota_client.post_groups(service_group, update=False)
+        self.iota_client.post_device(device=Device(**device), update=False)
+
+        clear_iot_agent(url=self.iota_url, fiware_header=self.fiware_header)
+
+        self.assertEqual(0, len(self.iota_client.get_device_list()) or len(self.iota_client.get_group_list()))
+
+    def test_clear_quantumleap(self):
+        # TODO
+        pass
```
Comment (sbanoeon) [inline tests/utils/test_clear.py:168]: Could you please add quantumleap test?

**Review by sbanoeon — CHANGES_REQUESTED** [review]

**Review by sbanoeon — CHANGES_REQUESTED** [review]
Also, please include changes in CHANGELOG.md.

**Comment by djs0109** [comment]
@sbanoeon could you check again after the tests runing through?

File: tests/utils/test_clear.py (hunk 1 above)
Comment (SystemsPurge) [reply tests/utils/test_clear.py]: I must be missing something, because i already see a quantumleap test

**Review by sbanoeon — APPROVED** [review]

**Pull request merged**